packages/idea-transformer/src/Transformer.ts

```diff
@@ -67,14 +70,16 @@ export default class Transformer<T extends Record<string, unknown>> {
             //make sure there is a schema model
             schema.model = schema.model || {};
             //loop through child types
-            for (const [ name, model ] of Object.entries(child.model)) {
+            for (const [name, model] of Object.entries(child.model)) {
               const parent = schema.model[name];
               //if type from child doesn't exist in schema (parent)
-              if (!parent) {
+              //I add !parent.mutable to make sure before adding to 
```

Comment:
Remove commit and code change notes. If someone were to randomly read this file, this would have no context. We should use notes to explain the current logic.